### ISSUE TITLE
Error in startup after #48

### ISSUE THREAD

**Issue by panosz** [jghauser/papis.nvim#50]
After pulling #48, i get the following error on nvim startup:
```
lazy.nvim  ERROR Failed to run `config` for papis.nvim
                                                                                                                 
...z/.local/share/nvim/lazy/papis.nvim/lua/papis/config.lua:163: attempt to concatenate local 'testing_conf_path'
 (a nil value)                                                                                                   
                                                                                                                 
# stacktrace:                                                                                                    
  - /papis.nvim/lua/papis/config.lua:163 _in_ **get_papis_py_conf**                                              
  - /papis.nvim/lua/papis/config.lua:236 _in_ **update**                                                         
  - /papis.nvim/lua/papis/init.lua:31 _in_ **setup**                                                             
  ...
```

**Comment by jghauser** [comment]
This should be fixed in [76bdb14](https://github.com/jghauser/papis.nvim/commit/76bdb14212c3b3b20b034b4775fe777569493ad5)!

**Comment by panosz** [comment]
It is! thanks very much!!!
